packages/playwright-core/src/utils/browserFetcher.ts

```diff
@@ -63,3 +66,63 @@ export function logPolitely(toBeLogged: string) {
   if (!logLevelDisplay)
     console.log(toBeLogged);  // eslint-disable-line no-console
 }
+
+let cachedUserAgent: string | undefined;
+function getDownloadUserAgent(): string {
+  if (cachedUserAgent)
+    return cachedUserAgent;
+  cachedUserAgent = determineDownloadUserAgent();
+  return cachedUserAgent;
+}
+
+function determineDownloadUserAgent(): string {
+  try {
+    const { identifier: osIdentifier, version: osVersion } = getOSInfo();
+    const userAgentTokens = [
+      getLanguage(),
+      osIdentifier,
+      osVersion,
+      os.arch(),
+      getAsBooleanFromENV('CI') ? '1' : '0',
+    ];
+    return `Playwright/${getPlaywrightVersion()} (${userAgentTokens.join('; ')})`;
```

Comment:
Let's follow the established UA practice and include Platform information (Node, Python, Java, .Net version)
```
Playwright/1.17.2 (amd64; macOS 10.15.7) Node/1.16.8
Playwright/1.17.2 (aarch64; macOS 12.0.1) Python/3.9
```

Also make sure params are easy to understand (you are using 1 for CI). Something like this:
```
Playwright/1.17.2 (amd64; macOS 10.15.7) Node/1.16.8 CI/1 (GitHubAction)
```

Otherwise in your user agent, `CI` is hard to decipher - I'm just getting `;1` and that means `CI`.

Reply:
Thinking out loud: Playwright/1.17.2 Python/3.9 macOS/12.0.1 (aarch64)